src/game/boe.global.hpp

```diff
@@ -31,4 +31,14 @@ extern std::map<std::string, int> startup_button_indices;
 extern std::map<int, std::string> startup_button_names;
 extern std::map<int, std::string> startup_button_names_v1;
 
+extern std::map<std::string, std::string> feature_flags;
+
+inline bool has_feature_flag(std::string flag) {
+	return feature_flags.find(flag) != feature_flags.end();
+}
+inline std::string get_feature_flag(std::string flag) {
+	if(!has_feature_flag(flag)) return "";
+	return feature_flags[flag];
```

Comment:
```suggestion
	std::map<std::string, std::string>::const_iterator iter = feature_flags.find(flag);
    if(iter == feature_flags.end()) return "";
	return *iter;
```

And the same in scenario.cpp.